projects/igniteui-angular/src/lib/grids/cell.component.ts

```diff
@@ -505,6 +505,7 @@ export class IgxGridCellComponent implements OnInit, OnChanges, OnDestroy {
 
     protected get selectionNode(): ISelectionNode {
         return {
+            cell: this,
```

Comment:
The cell instance's data may change when scrolling occurs. We should either  store the mrl specific coordinates of the cell or when navigation starts from group row:
- on ArrowUp - always move focus to the fist cell with the last rowStart in the current columnLayout
- on Arrow Down - always move focus to the fist cell with first rowStart in the current columnLayout